variables.tf

```
@@ -53,48 +53,49 @@ variable "customer_gateway_bgp_asn" {
 }
 
 variable "customer_gateway_ip_address" {
-  type        = "string"
+  type        = string
   description = "The IP address of the gateway's Internet-routable external interface"
 }
 
 variable "route_table_ids" {
-  type        = "list"
+  type        = list(string)
   description = "The IDs of the route tables for which routes from the Virtual Private Gateway will be propagated"
   default     = []
 }
 
 variable "vpn_connection_static_routes_only" {
-  type        = "string"
+  type        = string
   description = "If set to `true`, the VPN connection will use static routes exclusively. Static routes must be used for devices that don't support BGP"
   default     = "true"
 }
 
 variable "vpn_connection_static_routes_destinations" {
-  type        = "list"
+  type        = list(string)
   description = "List of CIDR blocks to be used as destination for static routes. Routes to destinations will be propagated to the route tables defined in `route_table_ids`"
   default     = []
 }
 
 variable "vpn_connection_tunnel1_inside_cidr" {
-  type        = "string"
+  type        = string
   description = "The CIDR block of the inside IP addresses for the first VPN tunnel"
   default     = ""
```

I found these throw errors if they're empty so I had to set them to `null`. Consumers shouldn't need to do that.

```suggestion
  default     = null
```